twilight-model/src/oauth/application.rs

```diff
@@ -30,13 +33,19 @@ pub struct Application {
     /// Settings for the application's default in-app authorization, if enabled.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub install_params: Option<InstallParams>,
+    /// Interactions endpoint URL for the app.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub interactions_endpoint_url: Option<String>,
     /// Name of the application.
     pub name: String,
     pub owner: Option<User>,
     pub primary_sku_id: Option<Id<OauthSkuMarker>>,
     /// URL of the application's privacy policy.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub privacy_policy_url: Option<String>,
+    /// Role connection verification URL for the app.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub role_connections_verification_url: Option<String>,
     #[serde(default)]
     pub rpc_origins: Vec<String>,
     pub slug: Option<String>,
```

Comment:
I would lean towards no, a deprecation is usually done to not break libs that already support something, since we never supported `summary` we shouldn't add it.